packages/browser/src/posthog-featureflags.ts

```diff
@@ -913,6 +913,10 @@ export class PostHogFeatureFlags {
             properties['$early_access_feature_name'] = feature.name
         }
 
+        if (stage) {
+            properties['$feature_enrollment_state'] = stage
```

Comment:
state, or stage?  Feels weird to have the variable be one but the property be anotherl

Reply:
Yeah, we should name it `stage` because that's what's in the UI.